<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TempCalc</class>
 <widget class="QDialog" name="TempCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>100</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QRadioButton" name="radioButton_pmi">
        <property name="text">
         <string>Pmi</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_pms">
        <property name="text">
         <string>Pms</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>100</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_air">
     <property name="title">
      <string>Air</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="le_air">
        <property name="placeholderText">
         <string>temp</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_calcAir">
        <property name="text">
         <string>Calc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="output_air">
        <property name="text">
         <string>Air capacity</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_fuel">
     <property name="title">
      <string>Fuel</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="le_fuel">
        <property name="placeholderText">
         <string>temp</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_calcFuel">
        <property name="text">
         <string>Calc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="output_fuel">
        <property name="text">
         <string>Fuel capacity</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="groupBox_fuel_2">
     <property name="title">
      <string>Mix</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLineEdit" name="le_capAir">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Air capacity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_capFuel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Fuel capacity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_calcMix">
        <property name="text">
         <string>Calc</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="output_mix">
        <property name="text">
         <string>Mix capacity</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
